src/Extension/FluentExtension.php

```diff
@@ -316,6 +321,20 @@ public function augmentDataQueryCreation(SQLSelect $query, DataQuery $dataQuery)
     }
 
     /**
+     * Get the localised table name
+     *
+     * @return string
+     */
+    public function getLocalisedTable()
+    {
+        $tableName = $this->owner->baseTable() . '_' . self::SUFFIX;
+
+        $this->owner->extend('updateLocalisedTable', $tableName);
```

Comment:
I'm not keen on adding extension points unless we had a solid use case for these.